lib/cors-anywhere.js

```diff
@@ -234,6 +237,7 @@ function getHandler(options, proxy) {
     requireHeader: null,            // Require a header to be set?
     removeHeaders: [],              // Strip these request headers.
     setHeaders: {},                 // Set these request headers.
+    maxAge: null,                   // If set, an Access-Control-Max-Age header with this value (in seconds) will be added.
```

Comment:
Done. Also updated the tests and readme to note that the default behaviour is now an Access-Control-Max-Age header set to 0.